_posts/batlopez/2016-02-03-NatsChap5Exerc.md

```diff
@@ -0,0 +1,11 @@
+--- 
+layout: post
+author: batlopez
+title: "Nat's Chapter 5 Exercises!"
+---
+Exercise 1: 
+<iframe src="https://trinket.io/embed/python/61d804b72e" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+Exercise 2:
+<iframe src="https://trinket.io/embed/python/f6bfabbb32" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+I'm embarassed to say I really struggled with this first exercise because I didn't realize break had to go at the end of the if statement. I don't understand why I can't put the if statement with the break after the statement evaluating whether x is an integer. Is that true all the time or am I missing something? Also, I kept trying to use x=int(x) before this current statement and wasn't sure why that doesn't work. In general, once I figured out that break should go at the end, that totally made logical sense. I don't really get why the other two things I mentioned don't work.
```

Comment:
Don't be embarrassed!  The struggle is real for all of us.

The reason you have to check for done first is that `int('done')`, which you call in the elif, will throw an error.  That error will then take you to the except block.  Make sense?  I think that may cover both of your questions